api/views.py

```diff
@@ -453,10 +453,21 @@ def put(self, request, schema, table):
 
     def validate_column_names(self, column_definitions):
         """Raise APIError if any column name is invalid"""
+        MAX_COL_NAME_LENGTH = 50
```

Comment:
I guess I would prefer to put those constants at the top of the file, not inside the function, but ok for now